Statistics.md

```diff
@@ -0,0 +1,85 @@
+<!--
+  - Licensed to the Apache Software Foundation (ASF) under one
+  - or more contributor license agreements.  See the NOTICE file
+  - distributed with this work for additional information
+  - regarding copyright ownership.  The ASF licenses this file
+  - to you under the Apache License, Version 2.0 (the
+  - "License"); you may not use this file except in compliance
+  - with the License.  You may obtain a copy of the License at
+  -
+  -   http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing,
+  - software distributed under the License is distributed on an
+  - "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  - KIND, either express or implied.  See the License for the
+  - specific language governing permissions and limitations
+  - under the License.
+  -->
+
+Parquet statistics definitions
+===
+This file contains the specification of all supported encodings.
+
+### Bloom filter statistics
+
+Supported Types: Integer
+
+The bloom filter is stored per block which is used to filter some blocks when reading the parquet file.
+
+#### Background for bloom filter
```

Comment:
I think it would be fine to refer to an external page for this background, but I have no problem leaving it in.